### PR TITLE
Kt filters add suffix support for float filters

### PR DESCRIPTION
**Pull request by santiagoballadares**

### CODE REVIEW

**Review by FlorianWendelborn — APPROVED** [review]

**Review by HansAuger — APPROVED** [review]

File: packages/documentation/pages/usage/components/filters.vue [hunk 1]
```diff
@@ -96,13 +111,17 @@ export default defineComponent({
 				isLoading: boolean
 			}
 			locale: Kotti.I18n.SupportedLanguages
+			prefix: Kotti.FieldNumber.Props['suffix']
```
Comment (carsoli) [inline packages/documentation/pages/usage/components/filters.vue:114]: `prefix: Kotti.FieldNumber.Props['prefix']`

File: packages/kotti-ui/source/kotti-filters/types.ts [hunk 2]
```diff
@@ -129,6 +129,7 @@ export namespace KottiFilters {
 			Common & {
 				operations: OPERATION[]
 				step: KottiFieldNumber.Props['step']
+				suffix: KottiFieldNumber.Props['suffix']
```
Comment (carsoli) [inline packages/kotti-ui/source/kotti-filters/types.ts:132]: the `prefix` seems to have already been supported but not part of this type

should you add `prefix` here as well?

**Comment by carsoli** [comment]
Feel free to merge after, I also approve @santiagoballadares

**Comment by carsoli** [comment]
Might publish tonight, so it's a good time to

File: packages/kotti-ui/source/kotti-filters/types.ts (hunk 2 above)
Comment (santiagoballadares) [reply packages/kotti-ui/source/kotti-filters/types.ts:134]: Float and Currency column types updated with prefix and suffix support.

**Review by carsoli — APPROVED** [review]

**Pull request merged**